code/modules/food_and_drink/ingredients.dm

```diff
@@ -180,6 +180,8 @@
 			new /obj/item/reagent_containers/food/snacks/deviledegg(get_turf(src))
 			new /obj/item/reagent_containers/food/snacks/deviledegg(get_turf(src))
 			qdel(src)
+		if (prob(25))
```

Comment:
For safety I'd move this conditional above the qdel(src). Probably would never cause a problem, but it just doesn't make sense to have code to be executed om an object after you have it slated for garbage collection.